tests/serde.rs

```diff
@@ -106,6 +367,20 @@ fn test_serde_de_primary_types() -> Result<(), Box<EvalAltResult>> {
     Ok(())
 }
 
+#[test]
+fn test_serde_de_integer_types() -> Result<(), Box<EvalAltResult>> {
+    assert_eq!(42_i8, from_dynamic(&Dynamic::from(42 as INT))?);
```

Comment:
```rust
(42 as INT).into()
```

is more efficient as `Dynamic::from` uses a large if-else statement to match different known types.

Reply:
But it's OK, I can fix these later.